python/requirements.txt

```diff
@@ -20,8 +20,8 @@ xdg
 
 # revng.daemon - tests
 pytest
-requests
-requests-toolbelt
```

Comment:
You sure no one else is using them? @dwarfyresearch is adopting them in [`revng import debug-info`](https://github.com/revng/revng/pull/222/files#diff-7bb7274fc8491ebe8492d383091e1be40cbd9cb8092f00feb6d6897148e70b43R21), what do we want to do there?

Reply:
Also, the existing ```download-pdb``` script already depends on this.

Reply:
Restored requests, we'll use it for the sync stuff while using aiohttp for async